app/views/admin/edition_images/index.html.erb

```diff
@@ -1,6 +1,7 @@
 <% content_for :context, "#{@edition.title}" %>
 <% content_for :page_title, "Images for #{@edition.format_name}" %>
 <% content_for :title, "Images for #{@edition.format_name}" %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @edition, parent_class: "edition")) %>
```

Comment:
Does this actually work?

I'm surprised you haven't needed to set `object: @new_image` rather than `object: @edition` 🤔

```suggestion
<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @new_image, parent_class: "edition")) if @new_image.present? %>
```

Reply:
Good spot, this was a hang up from changing the `edition_images_controller`. I've just updated it